app/services/sources/sources.ts

```diff
@@ -327,6 +329,21 @@ export class SourcesService extends StatefulService<ISourcesState> implements IS
       resolvedSettings.video_device_id = this.defaultHardwareService.state.defaultVideoDevice;
     }
 
+    // setup game capture
+    if (type === 'game_capture') {
+      // setup the list of supported games for the auto mode in the game_capture
+      resolvedSettings.auto_capture_list_path = path.join(
+        electron.remote.app.getPath('userData'),
+        'game_capture_list.lst',
+      );
+
+      // setup the placeholder image
+      resolvedSettings.auto_placeholder_image = path.join(
+        electron.remote.app.getAppPath(),
+        'media/images/game-capture/capture-placeholder.png',
```

Comment:
Pretty sure this needs to be `require`d to work with the app when it is packaged.